addons/argocd/disable

```diff
@@ -15,9 +15,18 @@ echo "Disabling ArgoCD"
 $HELM delete argo-cd -n $NAMESPACE_ARGOCD
 
 $KUBECTL delete customresourcedefinition applications.argoproj.io \
+                applicationsets.argoproj.io \
                 argocdextensions.argoproj.io  \
                 appprojects.argoproj.io
 
-$KUBECTL delete $KUBECTL_DELETE_ARGS namespace "$NAMESPACE_ARGOCD" > /dev/null 2>&1 || true
+if [[ "$1" == "--purge" ]]; then
```

Comment:
This looks very restrictive, if you are adding arguments to the script, maybe make sure that the command line is parsed properly, see for example https://github.com/canonical/microk8s-core-addons/blob/a47841e2292ca6a82c8a98057010630c4865cd63/addons/ha-cluster/disable#L13-L27